janitor/functions/read_commandline.py

```diff
@@ -0,0 +1,77 @@
+import pandas_flavor as pf
+import pandas as pd
+from subprocess import Popen, PIPE
+import sys
+
+
+@pf.register_dataframe_method
```

Comment:
I think this line isn't necessary, because the intent of `read_commandline` is to be a function that operates outside of a dataframe.